src/yargs-check.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+function reqArgsCheck(args) {
```

Comment:
In hindsight, I think it would be better if each argument ensures that it can't be the empty string. this encapsulates the validation to the argument.

eg:

```
      demandOption: true,
      requiresArg: true,
      coerce: (v) => (v.trim() ? v.trim() : undefined),
```

Reply:
I'm going to forego the demandOption attribute to keep with deploy; as the one-shot deploy doesn't necessitate it. I'll leave it up to every use of an option; to determine if that use should demand it